composer.json

```diff
@@ -66,12 +66,13 @@
     },
     "autoload": {
         "psr-4": {
-            "Sylius\\PriceHistoryPlugin\\": "src/",
-            "Tests\\Sylius\\PriceHistoryPlugin\\": "tests/"
+            "Sylius\\PriceHistoryPlugin\\": "src/"
         }
     },
     "autoload-dev": {
         "psr-4": {
+            "spec\\Sylius\\PriceHistoryPlugin\\": "spec/",
+            "Tests\\Sylius\\PriceHistoryPlugin\\": "tests/",
```

Comment:
If I'm thinking correctly, moving tests to dev autoload will mean that you won't be able to use our behat services (contexts etc) in the end application, so although this configuration makes sense and seems more correct, I don't know if it's recommended for our products